package.json

```diff
@@ -13,6 +13,7 @@
     "antd": "^4.4.3",
     "axios": "^0.19.2",
     "craco-less": "^1.17.0",
+    "dotenv": "^8.2.0",
```

Comment:
Does not have to be installed, react-scripts [installs it as a dependency](https://github.com/Lambda-School-Labs/merchant-marketplace-a-fe/blob/main/package-lock.json#L18300)

Reply:
Thank you! dotenv is removed from package.json.